### PR TITLE
fix(attachmentService, propfindErrorParse): add an error message for when a file is not compatible for windows

### PR DESCRIPTION
**Pull request by GVodyanov**
Fix https://github.com/nextcloud/calendar/issues/6245

Mostly copied from https://github.com/nextcloud/spreed/blob/main/src/utils/propfindErrorParse.ts

### CODE REVIEW

**Review by miaulalala — CHANGES_REQUESTED** [review]
feat, not enh :wink:

**Review by ChristophWurst — APPROVED** [review]

**Comment by miaulalala** [comment]
Does this need a backport to stable4.7 as well?

**Comment by ChristophWurst** [comment]
No because https://github.com/nextcloud/server/issues/44963 is for 30+

File: src/utils/propfindErrorParse.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+async function parseUploadError(exception) {
+	try {
+		const responseText = await exception.response.data
+		const parser = new DOMParser()
+		const xmlDoc = parser.parseFromString(responseText, 'application/xml')
+		const messageElement = xmlDoc.getElementsByTagName('s:message')[0]
+
+		return messageElement?.textContent
+	} catch (parseError) {
+		console.error(t('spreed', 'Error while parsing a PROPFIND error'), parseError)
```
Comment (st3iny) [inline src/utils/propfindErrorParse.js:22]: Should be sourced from `calendar` instead of `spreed`.

**Pull request merged**